src/platform/testing/e2e/cypress/support/commands/axeCheck.js

```diff
@@ -26,28 +26,16 @@ const processAxeCheckResults = violations => {
 
 /**
  * Checks the current page for aXe violations.
- *
- * @param {boolean} assert - Flag for how tests handle aXe violations.
- *   If true, tests will stop and fail when there are violations.
- *   If false, tests will skip over violations and continue running.
  */
-Cypress.Commands.add('axeCheck', (assert = true) => {
+Cypress.Commands.add('axeCheck', () => {
```

Comment:
It may be worth adding an optional parameter for changing the `context`. When I was writing `site-wide` tests, `.main` isn't a part of the DOM so I had to use `.checkA11y()` and pass in a different context.  This probably won't be used by many, but would still be nice to have just in case.

Reply:
Okay, that's a good point. Is there not even a 'main' element (even if the class isn't 'main')? What context worked there?

Reply:
There is a main element, which I had to use by calling `'main'` instead of `'.main'` inside of `.checkA11y()`. The `.main` class doesn't seem to exist on the homepage.

Reply:
Okay, I made the context an optional parameter that's `main` by default.

Reply:
I just tested it out, works like a charm!